ext/action-popup.html

```diff
@@ -11,94 +11,64 @@
-        <input type="checkbox" class="enable-search">
-        <div class="toggle-group">
-            <span class="toggle-on">On</span>
-            <span class="toggle-off">Off</span>
-            <span class="toggle-handle"></span>
         </div>
-    </label>
-    <a tabindex="0" class="link-group action-select-profile">
-        <span class="link-group-icon" data-icon="profile"></span>
-        <span class="link-group-label">Profile</span>
-        <span class="profile-select-container"><select class="profile-select">
-            <optgroup label="Default Profile" class="profile-select-option-group"></optgroup>
-        </select></span>
-    </a>
-    <a tabindex="0" class="link-group action-open-settings">
-        <span class="link-group-icon" data-icon="cog"></span>
-        <span class="link-group-label">Settings</span>
-        <span class="link-group-badge">
-            <div class="flex-margin-left warning-badge no-dictionaries-enabled-warning" hidden><span class="icon" data-icon="exclamation-point-short"></span></div>
-        </span>
-    </a>
-    <a tabindex="0" class="link-group action-open-permissions" hidden>
-        <span class="link-group-icon" data-icon="key"></span>
-        <span class="link-group-label">Permissions</span>
-        <span class="link-group-badge">
-            <div class="flex-margin-left warning-badge permissions-required-warning" hidden><span class="icon" data-icon="exclamation-point-short"></span></div>
-        </span>
-    </a>
-    <a tabindex="0" class="link-group action-open-search">
-        <span class="link-group-icon" data-icon="magnifying-glass"></span><span class="link-group-label">Search</span>
-    </a>
-    <a tabindex="0" class="link-group action-open-info">
-        <span class="link-group-icon" data-icon="question-mark-circle"></span><span class="link-group-label">Information</span>
-    </a>
-</div>
-</div>
-
+        <div tabindex="0" class="action-container">
+            <button type="button" class="low-emphasis action-open-settings" title="Settings" data-hotkey='["global:openSettingsPage","title","Settings ({0})"]'>
+                <div class="action-icon">
+                    <span class="icon" data-icon="cog"></span>
+                </div>
+            </button>
+            <button type="button" class="low-emphasis action-open-permissions" title="Permissions" hidden>
+                <div class="action-icon">
+                    <span class="icon" data-icon="key"></span>
+                    <div class="nav-button-warning-badge permissions-required-warning" hidden>
+                        <div class="nav-button-warning-badge-outer"></div>
+                        <div class="nav-button-warning-badge-inner"></div>
+                    </div>
+                </div>
+            </button>
+            <button type="button" class="low-emphasis action-open-search" title="Search" data-hotkey='["global:openSearchPage","title","Search ({0})\nShift+click to open here"]'>
```

Comment:
I'm not seeing this title anymore

<img width="200" alt="image" src="https://github.com/user-attachments/assets/a5d230e9-97de-4d29-a90d-07d7e0c7c579">